index.js

```diff
@@ -19,6 +19,11 @@ var connectionManager = require('./lib/connection-manager');
 var k8s = require('./lib/k8s');
 var utils = require('./lib/utils');
 
+// Variables
+var isWinAdmin; // windows only
+var processed;
```

Comment:
Personally I'd move this down closer to the block scope where it's used but up to you.